client/signup/steps/theme-selection/index.jsx

```diff
@@ -17,30 +18,28 @@ import Button from 'components/button';
 
 import { getSurveyVertical } from 'state/signup/steps/survey/selectors';
 
-const ThemeSelectionStep = React.createClass( {
-	displayName: 'ThemeSelection',
-
-	propTypes: {
+class ThemeSelectionStep extends Component {
+	static propTypes = {
 		designType: PropTypes.string,
 		goToNextStep: PropTypes.func.isRequired,
 		signupDependencies: PropTypes.object.isRequired,
 		stepName: PropTypes.string.isRequired,
 		useHeadstart: PropTypes.bool,
```

Comment:
Now that we are injecting it as a prop, we could add `translate: PropTypes.func` here.
And `translate = identity` in default props.